x-pack/metricbeat/_meta/kibana.generated/5/index-pattern/metricbeat.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
All the kibana.generated files should be under .gitignore

Reply:
gitignore added, thanks!